### DISCUSSION TITLE
F2010 Compset set up

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6697] [E3SM model help]
Hello,

I need to run E3SMv2.0 with the F2010 compset at equilibrium for one year. Should I set model_ start_type as initial, or, should I use hybrid. If hybrid, where can I find the appropriate initial/restart files?

Many thanks!

**Comment by whannah1** [comment]
Typical using "initial" will work fine with the default initial condition because the atmosphere only simulations do not require a long adjustment as long as a suitable spun-up land initial condition is used (which I think is set by default now). Using "branch" or "hybrid" is meant for starting with data from another run. 

When you say "equilibrium" do you have specific needs for this? Does it need to represent a specific time period? 
(this is largely influenced by the prescribed SST data - which is not easy to change)

**Reply by pgoddard08** [reply]
Hi Walter - thanks for your help.

After relaying your response to my colleagues, the initial run sounds like the right method. Equilibrium may not have been the best description, we just didn't know if the model would require years of spin-up. Am I correct that the F2010 would not require a long spin-up? - should I run for more than a year if I want to look at a year of output?

**Reply by whannah1** [reply]
Generally it's good to have a throw-away year of data if you can afford it - but the larger issue might be that 1-year of data makes it difficult to make any statistically conclusive statement. What are you expecting to be able to do with a single year of data?

**Reply by pgoddard08** [reply]
Good to know, thanks. Yes, we only need a year of data at the moment - it's for a machine learning test we are doing.
I appreciate your help

**Reply by whannah1** [reply]
Ah ok - in that case it probably doesn't matter what period you simulate. I suggest just running an F2010 case with all the defaults and using the first year of data.
